### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Upgrade to 5.4.0
- Apply example_ynh

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -83,49 +86,54 @@ ynh_app_setting_set --app=$app --key=final_path --value=$final_path
...
 #=================================================
 # PHP-FPM CONFIGURATION
 #=================================================
-ynh_script_progression --message="Configuring PHP-FPM..."
+ynh_script_progression --message="Configuring PHP-FPM..." --weight=2
 
 # Create a dedicated php-fpm config
-ynh_add_fpm_config --usage=low --footprint=low --package="$extra_php_dependencies"
+ynh_add_fpm_config --usage=low --footprint=low
+ynh_app_setting_set --app=$app --key=phpversion --value=$phpversion
```
Comment (yalh76) [inline scripts/install:108]: why setting `phpversion`, it's already set in ynh_add_fpm_config ?

File: scripts/install [hunk 2]
```diff
@@ -22,8 +22,10 @@ ynh_abort_if_errors
 
 domain=$YNH_APP_ARG_DOMAIN
 path_url=$YNH_APP_ARG_PATH
-admin_ampache=$YNH_APP_ARG_ADMIN
+admin=$YNH_APP_ARG_ADMIN
 is_public=$YNH_APP_ARG_IS_PUBLIC
+phpversion=$YNH_PHP_VERSION
```
Comment (yalh76) [inline scripts/install:27]: I don't think we need to do that

**Review by yalh76 — CHANGES_REQUESTED** [review]

**Comment by ericgaspar** [comment]
!testme

File: scripts/install (hunk 1 above)
Comment (yalh76) [reply scripts/install]: maybe, we will need `phpversion=$(ynh_app_setting_get --app=$app --key=phpversion)`

**Review by yalh76 — APPROVED** [review]

**Pull request merged**